plugins/security/resources/jenkins.groovy

```diff
@@ -215,16 +195,18 @@ class Actions {
         strategy.add(p,user)
       }
     }
-    if (perms.contains("gerrit")) {
-      for (Permission p : PluginImpl.PERMISSION_GROUP.getPermissions()) {
+    if (perms.contains("scm")) {
+      for (Permission p : SCM.PERMISSIONS.getPermissions()) {
         strategy.add(p,user)
       }
     }
-    if (perms.contains("scm")) {
-      for (Permission p : SCM.PERMISSIONS.getPermissions()) {
+
+    if (perms.contains("gerrit")) {
+      for (Permission p : GerritTriggerPluginImpl.PERMISSION_GROUP.getPermissions()) {
         strategy.add(p,user)
       }
```

Comment:
How about adding `if (perms.contains("all"))` block and corresponding schema changes?